custom/ewsghana/reports/__init__.py

```diff
@@ -61,6 +61,8 @@ class MultiReport(CustomProjectReport, CommtrackReportMixin, ProjectReportParame
     report_template_path = "ewsghana/multi_report.html"
     flush_layout = True
     split = True
+    exportable = True
+    is_exportable = False
```

Comment:
@czue I needed 3 states of this flag. When we select facility in location filter we should display facility page report which is always exportable. In other cases reports could, but not have to be exportable.
Also flag added by me works dynamically and depends on location filter and thanks to that I can hide/show export button and change text in it.